src/cpp/session/modules/SessionPythonEnvironments.R

```diff
@@ -235,6 +236,50 @@
    
 })
 
+.rs.addFunction("python.findPythonInterpretersInKnownLocations", function()
+{
+   # set of discovered paths
+   pythonPaths <- character()
+   
+   # Python root paths we'll search in
+   roots <- c(
+      "/opt/python",
+      "/opt/local/python",
+      "/usr/local/opt/python",
+      getOption("rstudio.python.installationPath")
```

Comment:
Is this documented anywhere?

Reply:
It is not; I added this mainly as a fallback to be used for testing. Perhaps this could be documented as a support article on Zendesk (covering Python integration with the IDE in general)?

Reply:
https://github.com/rstudio/rstudio/issues/8870